web/sass-files/sass/partials/_files.scss

```diff
@@ -133,12 +133,25 @@
     height: 100%;
     background-color: #FFF;
     background-repeat: no-repeat;
+    overflow: hidden;
+    position: relative;
     &.small {
       background-position: center;
     }
     &.normal {
       background-position: top left;
     }
+    .spinner.file__loading {
+        position: absolute;
+        left: 50%;
+        margin-left: -16px;
+        top: 50%;
+        margin-top: -16px;
+    }
+    .file__loaded {
+        height: 100px;
+        max-width: initial;
+    }
```

Comment:
![image](https://cloud.githubusercontent.com/assets/3277310/10579655/5007c16a-7647-11e5-92e8-111bf865afab.png)
Can we can size the gif differently when the image has an aspect ratio skinnier than the the attachment box? I uploaded a square gif and it has a static part of the image copied to fill the empty space. Wider gifs still work fine, but skinny ones have that gap on the right side.